geth/jail/internal/fetch/Makefile

```diff
@@ -1,5 +1,6 @@
-dist-fetch.rice-box.go: dist-fetch/bundle.js
-	rice embed-go
+generate: dist-fetch/bundle.js
+	go generate .
```

Comment:
I don't see any `//go:generate` comments in code.

Reply:
There is here https://github.com/status-im/status-go/pull/444/files#diff-85d1740e9cc6fe333147d0feb6046375R3

Reply:
Oh wow, missed it, thanks!